WcaOnRails/app/jobs/submit_results_nag_job.rb

```diff
@@ -0,0 +1,14 @@
+class SubmitResultsNagJob < ActiveJob::Base
+  queue_as :default
+
+  def nag_needed(competition)
+    (competition.results_nag_sent_at || competition.end_date) <= 1.week.ago
+  end
+
+  def perform(*args)
```

Comment:
These `*args` aren't used here, is it a convention to include them anyway in `perform`  methods?